NitroxPatcher/Patches/Dynamic/BulkheadDoor_OnPlayerCinematicModeEnd_Patch.cs

```diff
@@ -0,0 +1,28 @@
+using System.Reflection;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.Unity.Helper;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class BulkheadDoor_OnPlayerCinematicModeEnd_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((BulkheadDoor t) => t.OnPlayerCinematicModeEnd());
+
+    public static void Postfix(BulkheadDoor __instance)
+    {
+        if (!__instance.TryGetComponentInParent<NitroxEntity>(out NitroxEntity nitroxEntity, true))
```

Comment:
Is there a reason we are looking at the complete parent hierarchy? Where is the NitroxEntity component normally located?